src/shortcuts.jl

```diff
@@ -0,0 +1,85 @@
+"""
+    add_to!(a, b, c)
+
+Return the sum of `b` and `c`, possibly modifying `a`.
+"""
+add_to!(output, args...) = operate_to!(output, +, args...)
+
+"""
+    add!(a, b, ...)
+
+Return the sum of `a`, `b`, ..., possibly modifying `a`.
+"""
+add!(args...) = operate!(+, args...)
+
+"""
+    mul_to!(a, b, c)
+
+Return the product of `b` and `c`, possibly modifying `a`.
+"""
+mul_to!(output, args...) = operate_to!(output, *, args...)
+
+"""
+    mul!(a, b, ...)
+
+Return the product of `a`, `b`, ..., possibly modifying `a`.
+"""
+mul!(args...) = operate!(*, args...)
+
+"""
+    add_mul(a, args...)
+
+Return `a + *(args...)`. Note that `add_mul(a, b, c) = addmul(b, c, a)`.
```

Comment:
Shouldn't it be `add_mul(a, b, c) = muladd(b, c, a)`?

Reply:
Yes, indeed